OpenOversight/app/templates/cop_face.html

```diff
@@ -45,156 +45,159 @@ <h2>
-                  {% for error in form.dataWidth.errors %}
-                    <p>
-                      <span style="color: red;">[{{ error }}]</span>
-                    </p>
-                  {% endfor %}
-                  <div class="input-group input-group-sm">
-                    <input type="text"
-                           class="hidden form-control"
-                           name="dataHeight"
-                           id="dataHeight"
-                           placeholder="height">
-                  </div>
-                  {% for error in form.dataHeight.errors %}
-                    <p>
-                      <span style="color: red;">[{{ error }}]</span>
-                    </p>
-                  {% endfor %}
-                  <div class="input-group input-group-sm id-input">
-                    <label class="input-group-addon" for="officer_id">OpenOversight ID</label>
-                    <input type="text"
-                           class="form-control"
-                           name="officer_id"
-                           id="officer_id"
-                           placeholder="officer ID">
-                  </div>
-                  {% for error in form.officer_id.errors %}
-                    <p>
-                      <span style="color: red;">[{{ error }}]</span>
-                    </p>
-                  {% endfor %}
-                  <input type="text"
-                         class="hidden form-control"
-                         name="image_id"
-                         id="image_id"
-                         placeholder="image ID"
-                         value="{{ image.id }}">
-                  {% for error in form.image_id.errors %}
-                    <p>
-                      <span style="color: red;">Image: [{{ error }}]</span>
-                    </p>
-                  {% endfor %}
-                  <p>
-                    <div class="input-group addface-button">
-                      <input type="submit"
-                             class="btn btn-sm btn-primary form-control"
-                             value="Add identified face" />
-                    </div>
-                  </p>
-                  <div class="text-center input-explanation">
-                    <div class="text addface-button-explanation">
-                      <b>Explanation</b>: click this button to associate the selected image with the entered OpenOversight ID.
-                    </div>
-                    <div class="text id-explanation">
-                      <b>Explanation</b>: after matching the officer's name, badge number, or face to the roster, enter the officer's OpenOversight ID here.
-                    </div>
-                  </div>
-                </div>
+                </form>
+              {% else %}
+                <form action="{{ url_for('main.label_data', image_id=image.id) }}"
+                      method="post">
+                </form>
+              {% endif %}
```

Comment:
Found this bug while testing #1008 - it looks like we had some malformed HTML here (the <form> tag was never closed). The linting tool seems to have added in a closing tag here on its own which broke the face tagging tool

Reply:
I can address it in another PR!

Reply:
Where should the closing `</form>` tag go in this case?